pkg/version/verflag/verflag.go

```diff
@@ -85,10 +85,21 @@ func Version(name string, value versionValue, usage string) *versionValue {
 	return p
 }
 
+const (
+	versionFlagName = "version"
+	versionFlagHelp = "Print version information and quit"
+)
+
 var (
-	versionFlag = Version("version", VersionFalse, "Print version information and quit")
+	versionFlag = Version(versionFlagName, VersionFalse, versionFlagHelp)
 )
 
+func AddFlags(fs *flag.FlagSet) {
+	f := flag.Lookup(versionFlagName)
```

Comment:
how is this better than this:
```
func AddFlags(fs *flag.FlagSet) {
	fs.AddFlag(flag.Lookup(versionFlagName))
}
```

we're still pulling from and modifying global state... not seeing what the separated flag object gets us

Reply:
Instinct was "libs we control should offer flag registration helpers." But I think I agree it's not that much better in this case.

Reply:
Agree

Reply:
Done

Reply:
Done